apis/kueue/v1beta1/clusterqueue_types.go

```diff
@@ -97,6 +97,9 @@ type ClusterQueueSpec struct {
 	// +optional
 	AdmissionChecks []string `json:"admissionChecks,omitempty"`
 
+	// admissionCheckStrategy is the list of AdmissionChecks that should run when a particular ResourceFlavor is used.
```

Comment:
mention that this cannot be used in combination with `admissionChecks`. Add the same note for admissionChecks